lib/nexpose/scan_template.rb

```diff
@@ -176,6 +176,56 @@ def host_threads=(threads)
       host_threads.text = threads.to_s
     end
 
+    # Enable/disable IP stack fingerprinting
+    # @param [Boolean] enable or disable IP stack fingerprinting
+    def enable_ip_stack_fingerprinting(enable)
+      ns = REXML::XPath.first(@xml, 'ScanTemplate/Plugins/Plugin[@name="java/NetworkScanners"]')
```

Comment:
Changed max line width on master, so next time hound runs it shouldn't bark at these. ;)